deploy/deployment.yaml

```diff
@@ -3,10 +3,20 @@ apiVersion: v1
 metadata:
   name: registration-service
 objects:
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      labels:
+        provider: codeready-toolchain
+      name: registration-service
+      namespace: ${NAMESPACE}
   - kind: Deployment
     apiVersion: apps/v1
     metadata:
+      labels:
+        provider: codeready-toolchain
       name: registration-service
+      namespace: ${NAMESPACE}
     spec:
       replicas: ${{REPLICAS}}
```

Comment:
there is no need for `{{` here: `{` is enough (see https://docs.openshift.com/container-platform/3.11/dev_guide/templates.html#writing-parameters)

Reply:
we need `{{` because the value that is placed there is an integer with single curly braces it puts a string - I've already tried that ;-)

Reply:
ah right, I forgot about that 🤦‍♂